src/PrestaShopBundle/Resources/config/services/adapter/alias.yml

```
@@ -0,0 +1,29 @@
+services:
+  _defaults:
+    public: true
```

```suggestion
    public: false
```
Actually it's probably more efficient to set the default public to false here (and autowire to true)